container/build_tar.py

```diff
@@ -42,7 +42,7 @@ class DebError(Exception):
 
   @staticmethod
   def parse_pkg_name(metadata, filename):
-    pkg_name_match = TarFile.PKG_NAME_RE.match(str(metadata))
+    pkg_name_match = TarFile.PKG_NAME_RE.match(metadata)
     if pkg_name_match:
       return pkg_name_match.group('pkg_name')
     else:
```

Comment:
There are two problems, 
1. the regex wasn't matching (fixed)
2. line 49 returns base64.

this pr only fixes (1) as that is sufficient for our usecase.

I have no idea why the deb package name is base64, perhaps it's some temp file thing?